orderbook/openapi.yml

```diff
@@ -255,6 +255,20 @@ components:
       type: string
       example: "1234567890"
     FeeInformation:
+      description: |
+        Provides the information to calculate the fees.
+      type: object
+      properties:
+        expirationDate:
+          description: |
+            Expiration date of the offered fee. Order service might not accept
+            the fee after this expiration date. Encoded as ISO 8601 UTC.
+          type: string
+          example: "2020-12-03T18:35:18.814523Z"
+        amount:
+          description: Absolute amount of fee charged per order in specified sellToken
+          $ref: "#/components/schemas/TokenAmount"
+    LegacyFeeInformation:
```

Comment:
nit: I don't care that much, since u will kill this as soon as we are migrated. But you could just use the extend it https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism 

totally fine keeping it as u suggest

Reply:
I'd do it if LegacyFeeInformation directly extended the new FeeInformation. However, there we renamed min_fee to amount so I'd have to introduce a third type that both can extend, just to merge them into one again later.